### PR TITLE
https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/62

### PR DESCRIPTION
**Pull request by Olga-QA-USA**
Deleted comments

### CODE REVIEW

**Review by YanaLifanova — COMMENTED** [review]
Please format it . And add in describe and it numer of US. In comments add link to US

File: cypress/e2e/FreestyleProjectDeleteProjectOnProjectPage.cy.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        cy.get('.jenkins-button.jenkins-button--primary ').click();
+        cy.get('span').contains('new_Freestyle_project').should('not.exist');
+//         cy.get('dialog[open]').debug(); // Check if the dialog is present
+// cy.get('dialog[open] button[data-id="ok"]').debug().click(); // Debug the button before clicking
+// //
+    
+    //cy.get('dialog[open] button[data-id="ok"]').debug().should('be.visible').click()
+        //cy.get('button[data-id="ok"]').should('contain','yes').should('be.visible').click({ force: true });
+    });
+})
```
Comment (Negrescu29) [inline cypress/e2e/FreestyleProjectDeleteProjectOnProjectPage.cy.js:31]: @Olga-QA-USA are these comments required? as a best practice we shouldn't merge code that is commented out.

**Review by Negrescu29 — COMMENTED** [review]
I believe you shouldn't create new spec files for new test cases. 

if this is your TC number two for this US: US_01.004 | FreestyleProject > Delete Project it should go in the same spec file for this user story: cypress/e2e/freestyleProjectDelete.cy.js

so all TC for this US will come into the same spec file.

also check the commented out code

**Review by NShakiro — COMMENTED** [review]
I would suggest to update the branch, because we have a lot of new files created. Also, please fix describe and it names according to project rules

**Pull request closed without merging**